src/main/java/com/hackclub/hccore/slack/SlackBot.java

```diff
@@ -135,10 +206,53 @@ public SlackBot(HCCorePlugin plugin) throws Exception {
                 throw new RuntimeException(e);
               }
               return 1;
+            })).then(LiteralArgumentBuilder.<SlashCommandRequest>literal("lookup")
+            .then(RequiredArgumentBuilder.<SlashCommandRequest, String>argument("mention",
+                StringArgumentType.greedyString()).executes(context -> {
+              String mention = StringArgumentType.getString(context, "mention");
+              String id;
+              if (mention.startsWith("<@") && mention.endsWith(">")) {
+                int pipeIdx = mention.indexOf('|');
+                id = mention.substring(2, pipeIdx);
```

Comment:
Just want to check this has the correct behaviour with no pipe character. It returns -1, saying start from end, right?
Is it worth adding a comment explaining this